### DISCUSSION TITLE
Cannot find the refresh_token after updated APP

### DISCUSSION THREAD

**Discussion by timtim321123** [proshunsuke/colmsg#107] [General]
HI all, 

I cannot find the https://api.kh.glastonr.net/v2/signin for a refresh_token using mitmproxy.  Everyone has same matter?

My app version is 3.1.1 

App Updated Date :19/8/2024

**Comment by ranran1129** [comment]
I am facing the same problem and would like to know how to solve it.

**Comment by kt627** [comment]
I am facing the same problem.

**Comment by axlchr12** [comment]
I'm facing the same issue.

![image](https://github.com/user-attachments/assets/2fc909c4-3330-4a6f-acd3-b2efab3e395a)
![image](https://github.com/user-attachments/assets/cec58519-83e3-42ab-8574-3635e3ffe6f4)

I think it only captures the /update_token only, not /signin.
